demisto_sdk/tests/integration_tests/validate_integration_test.py

```diff
@@ -25,11 +25,11 @@
 AZURE_FEED_PACK_PATH = join(TEST_FILES_PATH, "content_repo_example/Packs/FeedAzure")
 AZURE_FEED_INVALID_PACK_PATH = join(TEST_FILES_PATH, "content_repo_example/Packs/FeedAzureab")
 VALID_PACK_PATH = join(TEST_FILES_PATH, "content_repo_example/Packs/FeedAzureValid")
-VALID_PLAYBOOK_FILE_PATH = 'demisto_sdk/tests/test_files/Packs/CortexXDR/Playbooks/Cortex_XDR_Incident_Handling.yml'
-INVALID_PLAYBOOK_FILE_PATH = 'demisto_sdk/tests/test_files/Packs/CortexXDR/Playbooks/' \
-                             'Cortex_XDR_Incident_Handling_invalid.yml'
-VALID_SCRIPT_PATH = 'demisto_sdk/tests/test_files/Packs/' \
-                    'CortexXDR/Scripts/EntryWidgetNumberHostsXDR/EntryWidgetNumberHostsXDR.yml'
+VALID_PLAYBOOK_FILE_PATH = join(TEST_FILES_PATH, "Packs/CortexXDR/Playbooks/Cortex_XDR_Incident_Handling.yml")
+INVALID_PLAYBOOK_FILE_PATH = join(TEST_FILES_PATH, "Packs/CortexXDR/Playbooks/Cortex_XDR_Incident_Handling_invalid.yml")
```

Comment:
as long as you are updating it to use join, please fix all of them...
```suggestion
INVALID_PLAYBOOK_FILE_PATH = join(TEST_FILES_PATH, 'Packs', 'CortexXDR', 'Playbooks', 'Cortex_XDR_Incident_Handling_invalid.yml')
```